zulipterminal/model.py

```diff
@@ -583,7 +582,7 @@ def update_stream_message(
         if content is not None:
             request["content"] = content
 
-        if self.server_feature_level is not None and self.server_feature_level >= 9:
+        if self.server_feature_level != 0 and self.server_feature_level >= 9:
```

Comment:
This can be simplified by writing a single conditional expression
(https://docs.python.org/3/reference/expressions.html#comparisons)